app/src/main/cpp/skyline/vfs/android_asset_filesystem.h

```diff
@@ -0,0 +1,28 @@
+// SPDX-License-Identifier: MPL-2.0
+// Copyright © 2021 Skyline Team and Contributors (https://github.com/skyline-emu/)
+
+#pragma once
+
+#include "filesystem.h"
+
+struct AAssetManager;
+
+namespace skyline::vfs {
+    /**
+     * @brief  The AndroidAssetFileSystem class provides the filesystem backing abstractions for the AAsset Android API
+     */
+    class AndroidAssetFileSystem : public FileSystem {
+      private:
+        AAssetManager *backing; //!< The backing manager of the filesystem
```

Comment:
Same as above, naming this `backing` rather than something like `assetManager` is very confusing as it can be conflated with our VFS backing abstraction